Asgardia.Constitution.Proposal.txt

```diff
@@ -116,21 +116,22 @@ Article 8. Key Citizen Rights and Freedoms 
+§ 7. All citizens have the right to seek legal reprieve in the Court of Justice for any infringement of rights and property. Every person may sue for redress as provided by law from the State or a public entity, in case he has suffered damage through the illegal acts of any public official. 
 § 8. Asgardian citizens may not be extradited to other states except as defined by the law.
 § 9. Asgardian citizens have the right to create associations and societies based on the law.
-§ 10. Citizen rights and freedoms may only be restricted by the law of Asgardia to the extent required to protect Asgardia’s national sovereignty, ensure Asgardia’s security, enable the performance of its mission and supreme values, and protect the rights of Asgardian citizens.
+§ 10. Citizen rights and freedoms may only be restricted by the Law of Asgardia to the extent required to protect Asgardia’s national sovereignty, ensure Asgardia’s security, enable the performance of its mission and supreme values, and protect the rights of Asgardian citizens.
 § 11. The enumeration of specific citizen rights and freedoms in this article shall not be interpreted as the negation or diminishment of other citizen rights and freedoms stipulated by other articles of Asgardia’s Constitution, the laws of Asgardia, and generally recognised human rights and freedoms.
+
 Article 9. Key Citizen Obligations
 § 1. Asgardian citizens have inalienable and indispensable rights. Asgardian citizen obligations arise from the moment they become citizens of Asgardia.
 § 2. All Asgardian citizens must respect, and not violate, the rights, freedoms, and established Law of Asgardia.
 § 3. All persons on Asgardian territory must comply with Asgardia’s Constitution, its laws and regulations, as well as respecting Asgardia’s Supreme Values.
-§ 4. All citizens of Asgardia must comply with the Constitution, Asgardia’s laws and regulations, respect and implement its Supreme Values irrespective of their location, unless this results in violation of the law of the country in which they are located.
+§ 4. All citizens of Asgardia must comply with Asgardia's Constitution, laws, and regulations, and respect and implement its Supreme Values irrespective of their location, unless this results in violation of the law of the country in which they are located.
 § 5. The people shall be liable to taxation as provided by law.
 § 6. Asgardian citizens have the right to participate in elections and referenda. No law, decree, or amendment shall prevent any citizen from participation in the processes of Government. Citizens regardless of origin, creed, or physical location shall be permitted to participate in elections and referenda.
-§ 7. Asgardian citizens have a responsibility to support good practices for conservation, regeneration and sustainability of all natural environments.

```

Comment:
I don't believe we have removed those protections at all. Rather they were moved to the more appropriate article where those protections made more sense.